packages/sandbox/src/components/ConnectButton/ConnectButton.tsx

```diff
@@ -18,7 +18,7 @@ export const ConnectButton = () => {
     try {
       /** Prompt the user to allow the snap */
       setLoading(true)
-      const response = await enableShapeShiftSnap(snapId)
+      const response = await enableShapeShiftMultichain(snapId)
```

Comment:
ditto, can LSP rename not to have to change this erwhere

Reply:
see abovesee above